### ISSUE TITLE
Tab Completion Beep is annoying

### ISSUE THREAD

**Issue by jakob** [jakob/Postico#513]
When pressing Tab to complete a word, Postico beeps if multiple completions are possible. Pressing Tab a second time shows completions.

(We wanted tab completion in Postico to be similar to tab completion in Bash.)

However, it seems that some people don't like the beep (myself included).

Does anybody like the beep?

**Comment by martinjankoehler** [comment]
We have been working on this issue!

Related changes:
- In case of ambiguity, possible suggestions are now shown directly without beeping.
- Bugfix: Inserting tab symbols after a tab-completed keyword works now
- Bugfix: Inserting tab symbol after "No Completions" works now

Download Build [B2491](https://eggerapps-downloads.s3.eu-west-1.amazonaws.com/postico-2491.zip), or check out the [list of recent builds](https://eggerapps.at/postico/builds/).

**Comment by deanmorin** [comment]
Hey guys! I tried out 2941, but I'm not sure how the new completion style is supposed to work. 

```sql
CREATE SCHEMA a;
CREATE TABLE a.foo (n integer);
CREATE TABLE a.foo_bar (n integer);
CREATE TABLE a.foo_foo (n integer);
```

If I start typing from the schema name, tab just inserts a tab character (`|` shows starting position):

`|a.` tab  --> `a.\t`
`|a.f` tab --> `a.f\t`

If the schema name is already there, it sort of does tab completion:

`a.|` tab --> `a.foo`

Sometimes it just picks one for you like vim completion, but without the ability to cycle through options. Other times it seems to sort of behave like bash completion. Sometimes tab cycles the completion window on and off.

Other than the bell, I much preferred the previous completion style.

**Comment by martinjankoehler** [comment]
@deanmorin Thanks for the feedback, this is a bug we have now fixed in the above [Build B2516](https://eggerapps-downloads.s3.eu-west-1.amazonaws.com/postico-2516.zip)

Tab will now only autocommit common prefixes whenever unambiguous.
In case of ambiguity, tab will yield a beep, but never commit the selected suggestion (this can be done with up/down/enter).
When initiating an autocomplete session via TAB, in case of ambiguity the initial beep is now intentionally gone, instead the list of alternatives suggestions is immediatly shown.

Please let us know in case of any other issues.

Thanks again, 
-- Martin

**Comment by jakob** [comment]
We've just released 1.3.3, with the improved tab completion. It should beep a lot less, now, and still behave similar to bash!

Thanks for all the feedback!

**Comment by deanmorin** [comment]
Works wonderfully now, thanks!
